packages/ng/simple-select/panel/panel.component.html

```diff
@@ -8,7 +8,11 @@
 	} as ctx"
 >
 	<div class="lu-picker-content" [class.is-loading]="loading$ | async" (scroll)="onScroll($event)">
-		<ng-container *luPortal="selectInput.panelHeaderTpl()" />
+		@if (selectInput.panelHeaderTpl(); as tpl) {
+		<div luCoreSelectPanelElement>
+			<ng-container *luPortal="tpl" />
+		</div>
```

Comment:
Should the whole header be a `luCoreSelectPanelElement` or it is up to components/templates in `panelHeaderTpl` to handle it?

Reply:
(former employee checkbox already call CoreSelectPanelElement)

Reply:
I feel like it should be inside the select-all indeed, for that case that is.

Reply:
I feel like it should be inside the select-all indeed, for that case that is.

Reply:
So it turns out the one in former employees is useless, and if each element provides its own `hostDirectives: [ɵCoreSelectPanelElement]`, it looks like the query isn't picking them, maybe due to them being added from a portal, I'm investigating further but wanted to report here first.

Reply:
Oh, because then the directive is deep and not in the base panel template, let's have some fun I guess 😄